cms/models_test.py

```diff
@@ -870,6 +876,27 @@ def test_external_course_page_properties():
     assert external_course_page.product == external_course_page.course
 
 
+def test_course_page_format_field_default_value():
+    """
+    Verifies that the "format" field in a course page has the default value FORMAT_ONLINE.
+    """
+    course_page = CoursePageFactory.create()
+
+    assert course_page.format == FORMAT_ONLINE
+
+
+def test_course_page_format_field_help_text():
+    """
+    Verify course page format field help text.
+    """
+    course_page = CoursePageFactory.create()
+
+    assert (
+        course_page._meta.get_field("format").help_text
+        == "A short description indicating the format of a program or course"
+    )
+
+

```

Comment:
We don't need a test for help_text fields or the field's description. 

A test that would be better is to create a couple of course pages:
1. With default value e.g. `Online`
2. With explicitly set value e.g. `Other`
3. Get the page and check that the context has the desired value.

[Here](https://github.com/mitodl/mitxpro/blob/master/cms/models_test.py#L120) is an example of a similar test.

Reply:
You will need to perform the same action for the program test written below

Reply:
Addressed